numpy/ma/tests/test_core.py

```diff
@@ -1723,6 +1723,17 @@ def setUp(self):
         xm[2] = masked
         self.intdata = (x, y, xm)
         self.floatdata = (x.astype(float), y.astype(float), xm.astype(float))
+        self.othertypes = (
```

Comment:
Nevermind, I guess that is included in `np.typecodes['AllFloat']`, but for what it is worth `np.typecodes['Complex']` would work for complex.